package io.apicurio.datamodels.core.diff.exceptions;public class RuleNotFoundException {
}
